src/pages/dashboard/charts/_Map.js

```diff
@@ -144,26 +128,37 @@ export default function Map({
     [maxValue],
   )
 
+  const isMobile = window.screen.availWidth < viewportSm
+
   return (
     <div className="map-container">
-      <div className="map-legend">
+      <div
+        className={['map-legend', useChoropleth ? 'choropleth' : 'bubble'].join(
+          ' ',
+        )}
+      >
         {useChoropleth ? (
           <ChoroLegend
             color={getColor[currentField]}
-            height={40}
-            width={300}
-            marginTop={8}
+            height={isMobile ? 180 : 36}
+            width={isMobile ? 32 : 300}
+            tickSize={isMobile ? 12 : 6}
             tickFormat="~s"
             spaceBetween={2}
-            tickSize={0}
           />
         ) : (
-          <BubbleLegend data={data} r={r} maxValue={maxValue} />
+          <BubbleLegend
+            data={data}
+            r={r}
+            maxValue={maxValue}
+            height={isMobile ? 100 : 150}
+            width={isMobile ? 100 : 150}
+          />
```

Comment:
I'm checking things in Chrome dev tools with iPhone X dimensions, and this element is falling offscreen to the right:

<img width="531" alt="Screen Shot 2020-04-13 at 5 08 27 PM" src="https://user-images.githubusercontent.com/63169602/79161420-75f29a80-7da9-11ea-9d7b-dff49883419a.png">

The legend seems less important for mobile. Wondering if we can:

- move it above or below the toggle, all elements stacked vertically
- move it below the map
- hide it entirely on small screens

Reply:
interesting, on my machine with iphone x dimensions this is what I see:
![image](https://user-images.githubusercontent.com/16123225/79180765-a1807f80-7dbf-11ea-9539-e71309491542.png)
I'm going to wait for the chart size changes from #527 to see how they play together before readdressing.